releasenotes/notes/instrumentation-openmetrics-8181dffaa1e6fb7f.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Cluster agent instrumentation has been switched from expvar to openmetrics
```

Comment:
Ah, my bad for commiting this, I guess we should move it to a cluster-agent release note folder?

Reply:
yeah will do